### PR TITLE
feat: Tailwind 2

### PR DESCRIPTION
**Pull request by esoubihe**
Updates Tailwind to version 2. Should only be released with https://github.com/Maxihost/metal-ui/pull/172 as we're now introducing custom colors to the lib.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,29 +40,29 @@
...
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "react-hot-loader": "^4.12.18",
-    "@emotion/babel-plugin-jsx-pragmatic": "^0.1.5",
-    "@emotion/react": "^11.1.5",
-    "@emotion/styled": "^11.1.5",
-    "@radix-ui/react-dropdown-menu": "0.0.10",
-    "babel-plugin-macros": "^3.0.1",
-    "twin.macro": "^2.3.0"
+    "react": "^16.14.0",
```
Comment (n23daniel) [inline package.json]: Shouldn't we still keep react & react-dom as peer dependency?

**Comment by esoubihe** [comment]
@n23daniel This is the PR that prepares the release. We still need to address the SSR issue we're getting

File: package.json [hunk 2]
```diff
@@ -26,6 +26,10 @@
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "7.10.4",
     "@babel/preset-react": "7.12.1",
+    "@emotion/babel-plugin-jsx-pragmatic": "^0.1.5",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.1.5",
```
Comment (n23daniel) [inline package.json:31]: I'll still look into the emotion issue on SSR before merging. We'd probably need to consider using [styled-components](https://github.com/styled-components/styled-components).

**Review by n23daniel — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (n23daniel) [reply package.json]: @esoubihe finally found the way to configure emotion for external usage. It's mainly based on the last reason from this comment: https://github.com/emotion-js/emotion/issues/1516#issuecomment-564302346 + tweaking the emotion babel plugins.
You can check it out in the latest commit - https://github.com/Maxihost/metal-ui/pull/173/commits/43d623fde90608ed5ed022981114276780ddc276

Comment (esoubihe) [reply package.json]: The project now fails to build for me. Am getting `Module not found: Error: Can't resolve '@emotion/styled/base`

Comment (esoubihe) [reply package.json]: Seems related to [this](https://stackoverflow.com/questions/65894711/module-not-found-error-cant-resolve-emotion-styled-base-when-running-story)

Comment (n23daniel) [reply package.json]: Thanks for finding the source of the issue. I fixed it here https://github.com/Maxihost/metal-ui/pull/173/commits/300e89f7a72e29e3fa1c9460637f238a679a21d2 by with manually linking to the correct modules until Storybook updates this. I subscribed to [this issue](https://github.com/storybookjs/storybook/issues/13277) to keep an eye for when they fix this.

**Pull request merged**